### PR TITLE
Advance controls - Wireframe mode

### PR DESCRIPTION
**Pull request by hey24sheep**
This pr aims to add advanced features to viewer.

**TODO**

- [x] Wireframe mode
 model

Issue tracking : #139

### CODE REVIEW

**Comment by kovacsv** [comment]
It's better to start pull requests for all the improvements separately to make it easier to discuss them.

**Comment by hey24sheep** [comment]
Hi, could you please check this PR too, thanks.

**Comment by kovacsv** [comment]
Thanks for the contribution, but I think this solution doesn't fit in the current user experience.

This kind of settings should be available from the settings panel, and not from the context menu. It also should be decided if we need to store the settings in cookies or not. Also, there is a question in case of embedding - will it be configurable or not. Moreover, it's needed to be configurable by the user: angle threshold and edge color. I understand that with the default settings it looks great with some models, but it should work with all kind of models.

To summarize this, to develop this feature it needs further discussion and design plans. I plan to do this later, but I'm currently working on other things.

**Comment by hey24sheep** [comment]
Okay, I understand.

**Comment by kovacsv** [comment]
I plan to implement it in the next release, so I close this one.
